build.gradle

```diff
@@ -374,6 +374,8 @@ dependencies {
     testImplementation "org.testcontainers:postgresql:1.6.0"
     testImplementation "com.h2database:h2"
     liquibaseRuntime "com.h2database:h2"
+    implementation "com.oracle.ojdbc:ojdbc8"
+    liquibaseRuntime "com.oracle.ojdbc:ojdbc8"
```

Comment:
why oracle dependency is added directly? In another ms - this dependency included dynamically on docker level.

Reply:
Ok, I will remove.
But at the same time other microservices have this dependency in gradle, for example loyalty.

Reply:
Better remove from another microservice also. Oracle JDBC driver requires additional licensing and not applied with A2.0 license.

Reply:
Ok, understood